<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowFileManager</class>
 <widget class="QMainWindow" name="MainWindowFileManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="m_tabWidget">
         <property name="elideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="m_labelStatus"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="m_horizontalSliderIconSize">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="m_progressBarFreeSpace">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="m_dockWidgetPlaces">
   <property name="windowTitle">
    <string>Places</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QListView" name="m_listViewPlaces"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_dockWidgetTerminal">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>38</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Terminal</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTermWidget" name="m_termWidgetTerminal">
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_dockWidgetFolders">
   <property name="windowTitle">
    <string>Folders</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QTreeView" name="m_treeViewFolders"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="m_dockWidgetPreview">
   <property name="windowTitle">
    <string>Preview</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_10">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGraphicsView" name="m_graphicsView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="m_toolBarTab">
   <property name="windowTitle">
    <string>Tabs</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddTab"/>
   <addaction name="actionCloseTab"/>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreateFile"/>
    <addaction name="actionCreateDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionAddTab"/>
    <addaction name="actionCloseTab"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionListView"/>
    <addaction name="actionTreeView"/>
    <addaction name="actionDetailsView"/>
    <addaction name="actionViewHide"/>
   </widget>
   <widget class="QMenu" name="menuGo">
    <property name="title">
     <string>Go</string>
    </property>
    <addaction name="actionGoPreviosDir"/>
    <addaction name="actionUpDir"/>
    <addaction name="separator"/>
    <addaction name="actionGoHome"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionTerminal"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuGo"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="m_toolBarNavigation">
   <property name="windowTitle">
    <string>Navigation</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionGoPreviosDir"/>
   <addaction name="actionGoFollowDir"/>
   <addaction name="actionUpDir"/>
   <addaction name="actionGoHome"/>
  </widget>
  <widget class="QToolBar" name="m_toolBarView">
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionIconView"/>
   <addaction name="actionListView"/>
   <addaction name="actionDetailsView"/>
   <addaction name="actionTreeView"/>
   <addaction name="actionViewHide"/>
  </widget>
  <widget class="QToolBar" name="m_toolBarZoom">
   <property name="windowTitle">
    <string>Zoom</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <action name="actionAddTab">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/document-open.png</normaloff>:/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Add Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionGoHome">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/home.png</normaloff>:/images/home.png</iconset>
   </property>
   <property name="text">
    <string>Go Home</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Home</string>
   </property>
  </action>
  <action name="actionGoPreviosDir">
   <property name="icon">
    <iconset theme="go-back" resource="../resources/resources.qrc">
     <normaloff>:/images/back.png</normaloff>
     <normalon>:/images/back.png</normalon>:/images/back.png</iconset>
   </property>
   <property name="text">
    <string>Previos Dir</string>
   </property>
  </action>
  <action name="actionCloseTab">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/window-close.png</normaloff>:/images/window-close.png</iconset>
   </property>
   <property name="text">
    <string>Close Tab</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionUpDir">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
   </property>
   <property name="text">
    <string>Up Dir</string>
   </property>
  </action>
  <action name="actionListView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons.png</normaloff>:/images/icons.png</iconset>
   </property>
   <property name="text">
    <string>List view</string>
   </property>
   <property name="toolTip">
    <string>List view</string>
   </property>
  </action>
  <action name="actionTreeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/bookmark.png</normaloff>:/images/bookmark.png</iconset>
   </property>
   <property name="text">
    <string>Tree view</string>
   </property>
  </action>
  <action name="actionDetailsView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/details.png</normaloff>:/images/details.png</iconset>
   </property>
   <property name="text">
    <string>Details view</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/zoom-in.png</normaloff>:/images/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/zoom-out.png</normaloff>:/images/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionViewHide">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/hidden.png</normaloff>:/images/hidden.png</iconset>
   </property>
   <property name="text">
    <string>View hide</string>
   </property>
   <property name="toolTip">
    <string>View hide</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionCreateFile">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/document-new.png</normaloff>:/images/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Create file</string>
   </property>
   <property name="toolTip">
    <string>Create file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionCreateDirectory">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/folder-new.png</normaloff>:/images/folder-new.png</iconset>
   </property>
   <property name="text">
    <string>Create directory</string>
   </property>
   <property name="toolTip">
    <string>Create directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="icon">
    <iconset theme="utilities-terminal" resource="../resources/resources.qrc">
     <normaloff>:/images/yakuake.png</normaloff>:/images/yakuake.png</iconset>
   </property>
   <property name="text">
    <string>Terminal</string>
   </property>
   <property name="toolTip">
    <string>Terminal</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="toolTip">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/system-file-manager.png</normaloff>:/images/system-file-manager.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionIconView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/icons.png</normaloff>:/images/icons.png</iconset>
   </property>
   <property name="text">
    <string>Icon view</string>
   </property>
   <property name="toolTip">
    <string>Icon view</string>
   </property>
  </action>
  <action name="actionGoFollowDir">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>Go follow dir</string>
   </property>
   <property name="toolTip">
    <string>Go follow dir</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTermWidget</class>
   <extends>QWidget</extends>
   <header>qtermwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
